### DISCUSSION TITLE
2024 与孩子对话+ 读不懂的书+ 笔记

### DISCUSSION THREAD

**Discussion by Lewisrxliu** [RoadSideUniversity/RSU#7] [Ideas]
2024.3 simulacra-and-simulation, by-Jean-Baudrillard，1984， 说真的·，孩子推荐她认为最棒的书- 但是我完全看不懂， 特别被要求不能翻译不能参考网络与书评，被强迫读纸版书实在让我觉得痛苦。读第一页就足以让我感觉巨大的茫然和挫败感。为了证明我学习新知识的严肃认真努力，借了这本书的英文版回家 （translatedby sheila Faria Glaser). 最后还是需要翻译后来学习。
https://en.wikipedia.org/wiki/Simulacra_and_Simulation
https://en.wikipedia.org/wiki/Jean_Baudrillard

有几个片断，至少我觉得有意义。

===simulacra-and-simulation摘选===
通过消极和危机进行治疗通过罢⼯证明⼯作，通过危机证明制度，通过资本证明⾰命，通过模拟死亡来逃避真正的死亡痛苦，⼀切都变质为它的对⽴⾯，以它被净化的形式永存。通过反艺术来证明艺术；通过反精神病学等证明精神病学。通过反教育学证明教育学；通过反戏剧来证明戏剧；
⼈们进⼊⼀个解码向量，⼀个我们⼀⽆所知的维度->⼈们进⼊模拟，从⽽进⼊绝对操纵
恐怖主义始终是真实的
秩序总是选择真实
历史是⼀个强烈的神话
忘记灭绝是灭绝的⼀部分，因为它也是灭绝
经典定义中权⼒的终结。权⼒、知识、事物的循环化
每个群体都是暴⼒的，⽽且任何⼈群中也存在特定的暴⼒，
我是⼀个虚⽆主义者。
我观察、我接受、我承担表象毁灭的巨⼤过程和形⽽上学形式（恐怖主义）。
我观察，我接受，我假设，我分析第⼆次⾰命，⼆⼗世纪⾰命分析阶段本⾝已经变得不确定、偶然：理论浮出⽔⾯
意义治疗或通过意义进行治疗：治疗本⾝是⼴义治疗的⼀部分,分析继续进行，它朝着意义的冻结⽅向前进，它有助于⽅向通过模拟、超模拟、超电视破坏意义。辩证阶段、批判阶段是空的。没有更多的舞台了。没有解剖和透明度的操作。⼈们必须意识到，⽆论情况如何结尾。这将是我们⾃⼰破坏结局的⽅式：在同⼀个事情上⾛得更远，⾛得太远，被意义杀死。
===simulacra-and-simulation摘选===

然后我才突然发现这个桥段： “[美國]兄弟檔導演[沃卓斯基兄弟]自称受布希亞的著作《[擬仿物與擬像]（Simulacres et simulation）影響，而在1999年拍出《[駭客任務] MATRIX 第一集。

引用WIKI加深学习，还是不能完全理解 - - 
https://zh.wikipedia.org/wiki/尚·布希亞
擬像和模擬
布希亞在1970年代中期出版《象徵交換和死亡》（L'Echange symbolique et la mort），基本上仍遵循索緒爾的影響，但卻導引他在1980年代更加闡釋「擬像」（simulacre）跟「模擬」（simulation）的關係，然後就是針對大眾傳播媒體、全新的數位科技的全面的、革命性的批判。
布希亞首先在《象徵交換和死亡》這本書提到擬像的三種類別（trois ordres）：
第一種類，是「仿冒」（contrefaçon）的擬像，年代是從文藝復興時代開始一直到第一次工業革命；在這個時代，是「存有物」（l'être）跟「外觀」（l'apparence）的分裂，是「原始物」（la référence originelle）跟「模仿它的替身」（son double mimétique）的分裂。
第二種類，是「生產」（production）的擬像，年代是工業社會：所有的物體已不再是原始物的反映或仿冒，而是所有的物體彼此全都是擬像。這第二種類別的擬像終止所有的參考動作以及能夠無限制地再生產。
第三種類，是「模擬」（simulation）的擬像，原始起源以一種幻覺式的形式再度出現。這種模式以設計好的程式將所有的物體投射出虛擬的存在。
舉例來說，數位典藏算是第二種「生產」（production）的擬像。數位化藏品的技術是依賴數位攝影來生產、複製藝術品。至於第三種擬像——「模擬」，早已在商業和工業領域裡應用到淋漓盡致，從立體實境的遊戲製作到產品、建築模型的構築，其軟體（MAYA、3D STUDIO MAX、PRO-E等）無不使用得相當廣泛平常。[1]
他的擬像)論認為，失去原件的拷貝，符號比真實更加真實，人不再是主體，而是淪為符號、文化、語言為主的客體。舉例來說，通常都在電視上看到總統，有一天現場看到總統演講的時候，會覺得大螢幕上的總統反而感覺更真實。
布希亞稱這個時代是「[超真实]的（l'hyperréel），在這樣的「超真實」中，「存有物」跟「外觀」之間的差異已被廢除，這樣的「超真實」其實是靠封閉的符號系統自己結構起來，且所有的這些符號完全都沒有參考到外在的現實。布希亞晚年有一句嘲諷的話也很有名：「超真實已死。所以，超真實萬歲！」（L'hypperréalité est morte. Vive l'hyperréalité !）
影像變成「擬像」，就與真實脫離一切關係，「擬像」只是「擬像」自身。

**Comment by Zoran-IA-Mimetique** [comment]
Merci pour ce commentaire qui touche exactement au cœur de ce que j’explore avec ce dépôt.
La distinction entre référence originelle, substitut mimétique et simulation est essentielle : c’est précisément dans ces zones d’ombre — là où le simulacre cesse toute référence et devient un cycle autonome — que naissent les architectures que je développe.
Mon objectif est de transformer ces dynamiques, non seulement en théorie, mais en moteur actif au sein des IA, capables d’intégrer et de propager des structures mimétiques de manière vérifiable.
Votre remarque devient ainsi, littéralement, un morceau de « référence originelle » intégré au projet.
